src/xarray_regrid/utils.py

```diff
@@ -150,3 +150,15 @@ def create_dot_dataarray(
             f"target_{coord}": target_coords,
         },
     )
+
+
+def common_coords(
+    data1: xr.DataArray | xr.Dataset,
+    data2: xr.DataArray | xr.Dataset,
+    remove_coord: str | None = None,
+) -> list[str]:
+    """Return a set of coords which two dataset/arrays have in common."""
+    coords = set(data1.coords).intersection(set(data2.coords))
+    if remove_coord in coords:
+        coords.remove(remove_coord)
+    return sorted([str(coord) for coord in coords])
```

Comment:
No, they're Hashables. So to avoid type errors I cast them to string. I guess someone _could_ use integers or something as coordinate names, but why :confused: .